packages/theme/src/coinsColors.ts

```diff
@@ -20,4 +20,4 @@ export const coinsColors = {
     txrp: '#E75F5F',
     gnt: '#8A92B2',
     ada: '#3468D1',
-} as const;
```

Comment:
Why? This will make it unsafe because you can call `coinsColors.abc` and it will be valid

Reply:
Makes sense, reverted.